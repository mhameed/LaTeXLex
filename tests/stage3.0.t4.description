author: ehc
date: 09/03/23
description: tests that the right amount of whitespace is left in the file in response to various items that would usually create visual spacing (newlines)

This was failing and now is not but I don't appear to have dealt with the below comment. 

Once that is fixed, remove the easy case of the fail at the top and then duplicate the whole content with:
* no newlines either side of the things such as \medskip
* whitespace either side of the things such as \medskip
* as it is now but with broken lines inside the paragraphs

These should all produce the same output but currently don't, this might be fixed by the same fix as earlier but this should be checked. I suspect there is a second bug there