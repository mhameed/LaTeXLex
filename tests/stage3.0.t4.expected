A paragraph

A second

A third

A paragraph 

This is a new paragraph following a par and should not be pulled up

Now we have another paragraph, this followed a medskip and should not have been pulled up.


Now we have another paragraph, this followed a bigskip and should not have been pulled up.



Now we have a new page this should not have been pulled up.



Now we have another new page, this should not have been pulled up.

A new paragraph after a line of whitespace

A last paragraph after a line of spaces and tabs 
